components/ViewItem.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import { v4 as uuidv4 } from "uuid";
+import Link from "next/link";
+import { myViewsContext } from "@/pages/_app";
+import { useContext } from "react";
+
+export default function ViewItem() {
+  const globalViews = useContext(myViewsContext);
+
+  return (
+    <Ul>
+      {globalViews.map((view) => (
+        <Li key={uuidv4()}>
+          <Link href={`/project-details/view-details/${view.slug}`}>
+            {view.name}
+          </Link>
+        </Li>
+      ))}
+    </Ul>
+  );
+}
+
+const Ul = styled.ul`
```

Comment:
components with such a short name of `UL` are hard to read and not as advisable.